pyproject.toml

```diff
@@ -1,7 +1,88 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel"]
+requires = ["setuptools>=49.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "exspy"
+description = "EELS and EDS analysis with the HyperSpy framework"
+requires-python = ">=3.8"
+readme = "README.md"
+keywords=[
+    "data analysis",
+    "microscopy",
+    "electron microscopy",
```

Comment:
Explicitly add the two main methods "electron energy loss spectroscopy" and "energy-dispersive X-ray spectroscopy" as keywords? Possibly also the abbreviations EELS and EDS and/or EDX.